config/initializers/apipie.rb

```diff
@@ -113,3 +113,30 @@ def description
         "dot(.), space, underscore(_), hypen(-) with no leading or trailing space."
   end
 end
+
+class AnyOfValidator < Apipie::Validator::BaseValidator
```

Comment:
I'm not sure I understand the difference between `AnyOfValidator` and `EnumValidator` from Apipie. Probably worth documenting the difference as a comment here.

Reply:
I changed the name to `AnyOfTypesValidator` and added a comment. The difference is that `Enum` validates param values, while this enumerates supported types. See the redmine issue for more details.